queries.sql

```diff
@@ -14,4 +14,79 @@ SELECT * FROM animals WHERE neutered = true;
 
 SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg  >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg  >= 10.4 AND weight_kg <= 17.3;
+
+/* Part 1 */
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+/* Part 2 */
+
+BEGIN;
+
+UPDATE animals
+SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
+
+/* Part 3 */
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
```

Comment:
- [ ] you should verify if the changes were made before and after rollback as well

Reply:
<img width="580" alt="3" src="https://user-images.githubusercontent.com/53530780/189985489-5c8d8e9f-2eff-4c8e-ad2d-1ae90da5f6b9.png">